docs/PREACT_DISPLAYING_DATA.md

```diff
@@ -267,6 +279,151 @@ class Result extends Component {
 }
 ```
 
+
+### Variants
+When variants are enabled, each result (in controller.store.results) contains a variants object with the following properties:
+
+`variants.active` - The currently selected variant's data.
+
+`variants.data` - Array of all variant data.
+
+`variants.selections` - Contains variant selection data (see below).
+
+
+and the following functions:
+
+`variants.setActive` - Function to update the selected variant.
+
+`variants.update` - Updates the variant data (see below for more details).
+
+`variants.makeSelections` - For internal usage only.
+
+`variants.refineSelections` -  Updates available variant options based on current selections to prevent invalid combinations
```

Comment:
Should this be mentioned as internal usage? I think typical usage would only be to interact with the selections eh?

Reply:
no I believe its just for internal use only. The normal users interact with the selections with selections.select function.